### PR TITLE
Fixing to the runtime Bi-Linear Resize.

### PR DESCRIPTION
**Pull request by yoya**
Fixing to the runtime Bi-Linear Resize.
Modified according to behavior of Pillow-6.0.0.

## Motivation and Context

You can resize with Bi-Linear filter, same as Pillow.

## Description

Resemble the behavior of the following code of PIL.

```python
from PIL import Image
im = Image.open(sys.argv[1])
im = im.resize([4, 4], Image.BILINEAR)
im.save(sys.argv[2])
```

## How has this been tested?

```
% cd blueoil/runtime/build
% DLK_LIB_DIR=`pwd`/../examples/dlk_lib/ cmake -DBUILD_SHARED_LIBS=ON ../
% make test
```

Test array data(8x8 => 4x4 conversion) was generated by the following python program.

- PIL_resize_BILINEAR_test.py 
  - https://gist.github.com/yoya/519b67e158fa5a4fd028f50f72123b3a

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: runtime/src/blueoil_image.cpp [hunk 1]
```diff
@@ -100,26 +100,39 @@ Tensor ResizeHorizontal_BiLinear(const Tensor &tensor, const int width) {
   const int channels  = shape[2];
   const int height = srcHeight;
   Tensor dstTensor({height, width, channels});
-  float xScale = static_cast<float>(width) / static_cast<float>(srcWidth);
-  int xSrcWindow = std::floor(1/xScale);
-  xSrcWindow = (xSrcWindow < 2)? 2 :xSrcWindow;
+  const float xScale = static_cast<float>(width) / static_cast<float>(srcWidth);
+  const float xSrcWindow = (xScale < 1.0)?(1.0f / xScale):1.0;
```
Comment (tkng) [inline runtime/src/blueoil_image.cpp]: Could you insert a space before/after `?` and `:`?

File: runtime/src/blueoil_image.cpp [hunk 2]
```diff
@@ -100,26 +100,39 @@ Tensor ResizeHorizontal_BiLinear(const Tensor &tensor, const int width) {
...
+            const float *srcPixel = tensor.dataAsArray({srcY, srcX, c});
+            float w = 1.0 - d;  // Bi-Linear
+            v += w * (*srcPixel);
+            totalW += w;
+          }
         }
-        float *dstRGB = dstTensor.dataAsArray({dstY, dstX, 0});
-        dstRGB[c] = v / totalW;
+        float *dstPixel = dstTensor.dataAsArray({dstY, dstX, c});
+        *dstPixel = (v)?(v/totalW):0;
```
Comment (tkng) [inline runtime/src/blueoil_image.cpp]: Could you insert a space before/after ? and :?

File: runtime/src/blueoil_image.cpp [hunk 3]
```diff
@@ -133,26 +146,39 @@ Tensor ResizeVertical_BiLinear(const Tensor &tensor, const int height) {
   const int channels  = shape[2];
   const int width = srcWidth;
   Tensor dstTensor({height, width, channels});
-  float yScale = static_cast<float> (height) / static_cast<float>(srcHeight);
-  int ySrcWindow = std::floor(1/yScale);
-  ySrcWindow = (ySrcWindow < 2)? 2 :ySrcWindow;
-  for (int dstY = 0 ; dstY < height ; dstY++) {
-    for (int dstX = 0 ; dstX < width ; dstX++) {
+  const float yScale = static_cast<float> (height) / static_cast<float>(srcHeight);
+  const int ySrcWindow = (yScale < 1.0)?(1.0f / yScale):1.0;
```
Comment (tkng) [inline runtime/src/blueoil_image.cpp]: Could you insert a space before/after ? and :?

File: runtime/src/blueoil_image.cpp [hunk 4]
```diff
@@ -100,26 +100,39 @@ Tensor ResizeHorizontal_BiLinear(const Tensor &tensor, const int width) {
   const int channels  = shape[2];
   const int height = srcHeight;
   Tensor dstTensor({height, width, channels});
-  float xScale = static_cast<float>(width) / static_cast<float>(srcWidth);
-  int xSrcWindow = std::floor(1/xScale);
-  xSrcWindow = (xSrcWindow < 2)? 2 :xSrcWindow;
+  const float xScale = static_cast<float>(width) / static_cast<float>(srcWidth);
+  const float xSrcWindow = (xScale < 1.0)?(1.0f / xScale):1.0;
   for (int dstY = 0 ; dstY < height ; dstY++) {
```
Comment (tkng) [inline runtime/src/blueoil_image.cpp]: Could you delete the space to the left of the `;`?
The code will be like  `for (int dstY = 0; dstY < height; dstY++) {`.

Also, could you make the same modification for all` for` loops below?

File: runtime/src/blueoil_image.cpp (hunk 1 above)
Comment (yoya) [reply runtime/src/blueoil_image.cpp]: https://github.com/blue-oil/blueoil/pull/611/commits/5dd6c9aee43af40bfbf32b738d2b0ffb07631947#diff-37230609db7212ca72dd52076761c190L104

File: runtime/src/blueoil_image.cpp (hunk 2 above)
Comment (yoya) [reply runtime/src/blueoil_image.cpp]: https://github.com/blue-oil/blueoil/pull/611/commits/5dd6c9aee43af40bfbf32b738d2b0ffb07631947#diff-37230609db7212ca72dd52076761c190L135

File: runtime/src/blueoil_image.cpp (hunk 3 above)
Comment (yoya) [reply runtime/src/blueoil_image.cpp]: https://github.com/blue-oil/blueoil/pull/611/commits/5dd6c9aee43af40bfbf32b738d2b0ffb07631947#diff-37230609db7212ca72dd52076761c190L151

File: runtime/src/blueoil_image.cpp (hunk 4 above)
Comment (yoya) [reply runtime/src/blueoil_image.cpp]: https://github.com/blue-oil/blueoil/pull/611/commits/5dd6c9aee43af40bfbf32b738d2b0ffb07631947

**Pull request merged**